controllers/apply-next/simple/fields.test.js

```diff
@@ -248,7 +248,7 @@ describe('fields', () => {
         });
     });
 
-    function assertRequiredForOrganistionTypes(field, requiredTypes) {
+    function assertRequiredForOrganisationTypes(field, requiredTypes) {
```

Comment:
Thanks for fixing this but FYI I've just removed this method as it's not used.

(Hopefully my move to Webstorm will remove all future typos…)